doc/source/style.rst

```diff
@@ -0,0 +1,25 @@
+.. _style:
+
+.. currentmodule:: pandas
+
+.. ipython:: python
+   :suppress:
+
+   import numpy as np
+   import pandas as pd
+   np.set_printoptions(precision=4, suppress=True)
+   pd.options.display.max_rows = 15
+
+*******
+Styling
+*******
+
+.. versionadded:: 0.17.1
+
+
```

Comment:
Working on that and the api.rst right now. My current plan is to included a step in `doc/make.py` to use nbconvert to convert the example notebook to HTML and host that on `pandas.pydata.org`. Sound reasonable?